src/components/CircleDetails.vue

```diff
@@ -192,18 +192,15 @@ export default {
 			}
 		},
 
-		onDisplayNameChangeDebounce: debounce(function(event) {
-			this.onDisplayNameChange(event.target.value)
+		onNameChangeDebounce: debounce(function(event) {
+			this.onNameChange(event.target.value)
 		}, 500),
-		async onDisplayNameChange(description) {
-			this.loadingDescription = true
```

Comment:
Right, probably a copy/paste leftover :)
Would you mind adding a loading indicator next to the title then?

Reply:
Sure, I could do that. But that would have to go to component `DetailsHeader`, as that's the one used to display the circle name field. You don't have a loading indicator there yet, not for contacts title either.

Shall I add a generic loading indicator to `DetailsHeader` and make `CircleDetails` use that one? If so, where would you place it? Left to the title/name input field looks very odd on big screens as the input field takes the full width per default. Another option would be to display it instead of (or as overlay over) the avatar.

Anyway, probably that change should go into a separate PR, no?

Reply:
I certainly meant "right to the title/name input field looks very odd" above :wink:

Reply:
I would just use the `title` slot as well

Reply:
Ok, that's the least invasive solution :)

I'm a bit unsure where to place the loading spinner design-wise. Just show it instead of the name field? Or in an overlay over the name field and place it there on the left?

Reply:
show it after the input?

Reply:
Ok, after playing around a bit and realizing that I don't know a better solution, I ended up adding the loading indicator after the input field now :)